Header  header       # timestamp

float64 voltage      # the voltage of the device, measured in Volt
float64 weight       # the power consumed in a day, measured in Watts
float64 current      # the current of the device at the moment, measured in Amp

int32   minutes_day  # he duration that the device has been used during a day, measured in minutes
